hijack/conf.py

```diff
@@ -0,0 +1,12 @@
+from appconf import AppConf
```

Comment:
what does this AppConf actually do in contrast to django AppConfig?

Reply:
It is a settings container, which allow you to define app specific settings including defaults. However, I will remove it. It's better to have no dependencies.